src/bin/mdbook-xgettext.rs

```diff
@@ -109,12 +191,49 @@ fn main() -> anyhow::Result<()> {
         .ok_or_else(|| anyhow!("Missing output.xgettext.pot-file config value"))?
         .as_str()
         .ok_or_else(|| anyhow!("Expected a string for output.xgettext.pot-file"))?;
+    // The depth on which to split the output file. 0 (default) is to include
+    // all messages into a single POT file, 1 will include each top-level file
+    // in a separate POT file, greater numbers will further split the main POT
+    // file.
+    let split_depth = cfg
+        .get("depth")
+        .unwrap_or(&Value::Integer(0))
+        .as_integer()
+        .ok_or_else(|| anyhow!("Expected a number for output.xgettext.depth"))?;
```

Comment:
A small trick which allows us to avoid depending directly on the `toml` crate:

```suggestion
    let split_depth = cfg
        .get("depth")
        .and_then(|value| value.as_integer())
        .ok_or_else(|| anyhow!("Expected a number for output.xgettext.depth"))?;
```

It turns out that mdbook depends on a slightly old version of the toml crate (see https://github.com/rust-lang/mdBook/issues/2019 for why), so it feels ever-so-slightly better if we can avoid mentioning this version in our own `Cargo.toml` file.